src/lib/markbind/src/patches/htmlparser2.js

```diff
@@ -95,20 +99,35 @@ var i = 0,
     IN_NUMERIC_ENTITY         = i++,
     IN_HEX_ENTITY             = i++, //X
 
-    j = 0,
+	SPECIAL_NONE              = 0,
 
-    SPECIAL_NONE              = j++,
-    SPECIAL_SCRIPT            = j++,
-    SPECIAL_STYLE             = j++;
+	/*
+	 Non parser-state constants
+	 */
+
+	// processSpecialFunctions, processSpecialClosingTagCharacter
+	HAS_MATCHING              = -1,
```

Comment:
Do you think it is possible to follow the previous enumerations for these

Reply:
Needed negative numbers here as the positive ones are 'taken' by the indexes